lib/page/missing_country.rb

```diff
@@ -0,0 +1,8 @@
+module Page
+  class MissingCountry
+    attr_reader :missing
+    def initialize(country, world)
+      @missing = world[country.to_sym]
```

Comment:
We thought about that as well, but the main layout template uses `@country` to display the headings (the name of the country, when you hit the country route), accessing `:name` and `:url`, but the missing country page doesn't display the country name (just the headings. We had to change that layout so that it could use the `@page.country` if it is available, i.e., for instances of a country page, but that would mean it would also use it when we send an instance of a missing country page... which is the reason why I think the old code contained `missing` instead of country.

We tried:
```erb
<% @country    = @page.country if (@page and @page.respond_to? :country and !@page.country.has? 'displayName' ) %>
```

in the old PR but decided to remove it because we were already feeling dirty enough with all those conditionals to add more on top of them, maybe we can revisit that line in this PR.

I think probably there is maybe too much logic in the layout template were the handling of the headings takes place, we could raise an issue to clean it.